docs/package.json

```diff
@@ -68,10 +69,20 @@
   },
   "babel": {
     "presets": [
-      "babel-preset-react",
-      "babel-preset-es2015",
-      "babel-preset-stage-1"
-    ]
+      "react",
+      "es2015",
+      "stage-1"
+    ],
+    "env": {
+      "production": {
+        "presets": [
+          "react",
+          "react-optimize",
+          "es2015",
+          "stage-1"
+        ]
+      }
+    }
```

Comment:
You can just add "bable-preset-react-optimize" in production, it will be merged with the defaults